ci/cases/pr/C48mx500_hybAOWCDA.yaml

```diff
@@ -0,0 +1,27 @@
+experiment:
+  system: gfs
+  mode: cycled
+
+arguments:
+  pslot: {{ 'pslot' | getenv }}
+  app: S2S
+  resdetatmos: 48
+  resensatmos: 48
+  resdetocean: 5.0
+  comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
+  expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
+  #icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C48mx500/20240610
+  icsdir: /work/noaa/da/gvernier/ensda/ictest
+  #icsdir: /scratch2/NCEPDEV/ocean/Guillaume.Vernieres/data/prepics/ictest/
```

Comment:
remove comments (2x)

Reply:
added a todo, waiting for the IC's to be staged before I remove the comments.